types/block.go

```diff
@@ -1048,41 +1048,37 @@ func (data *Data) Hash() tmbytes.HexBytes {
 	return data.hash
 }
 
-type Messages struct {
-	MessagesList []Message `json:"msgs"`
-}
-
 // ByNamespace implements sort.Interface for Message
-type ByNamespace []Message
+type Blobs []Blob
```

Comment:
the above `// ByNamespace implements sort.Interface for Message` comment needs to be removed I guess